lib/fldabnum.gd

```diff
@@ -314,7 +314,7 @@ BindGlobal( "CYCLOTOMIC_FIELDS",
             return AbelianNumberFieldByReducedGaloisStabilizerInfo( Rationals,
                    xtension, [ 1 ] );
         end,
-    [ Rationals,,, GaussianRationals ] )
+    [ Rationals, Rationals,, GaussianRationals ] )
```

Comment:
Your last commit "Add Rationals as the 2nd cyclotomic field" should be folded into the 4th, "Convert CYCLOTOMIC_FIELDS to use MemoizePosIntFunction"

Reply:
Why? Adding Rationals as the 2nd field is a separate change, which could have happened years ago.

Reply:
Let me rephrase: This change (adding Rationals as the 2nd cyclotomic field) should be done in a single commit. Right now, you do it for the HPC-GAP version of the file in your 4th commit, and for the GAP version in another commit.